apps/holidays/holidays.star

```diff
@@ -22,7 +29,9 @@ def main(config):
         "Dec 31": CONFETTI,
     }
 
-    now = time.now()
+    location = json.decode(config.get("location") or DEFAULT_LOCATION)
```

Comment:
Another option is to use the (undocumented and magic) `$tz` variable. It'll get passed in config when your app is running in the tidbyt production environment, and holds the device's selected timezone.

Reply:
Nice, that's better